src/main/java/dukeApp/files/Deadline.java

```diff
@@ -0,0 +1,94 @@
+package dukeApp.files;
+
+public class Deadline extends Task {
+    protected String word;
+    protected String date;
+    protected String time;
+    protected String dateLine;
+    protected String originalDate;
+
+    public Deadline(String s) {
+        super(s);
+        description = s.split("/")[0];
+        date = s.split("/")[1];
+        word = date.substring(0, date.indexOf(" "));
+        dateLine = date.substring(date.indexOf(" ") + 1, date.length());
+        date = dateLine.split(" ")[0];
+        time = dateLine.split(" ")[1];
+        originalDate = date;
+        date = changeDate();
+        time = changeTime();
+    }
+
+    /**
+     * Returns the type of task
+     *
+     * @return the letter indicating the task type
+     */
+    public String getType() {
+        return "D";
+    }
+
+    /**
+     * Returns the task date description
+     *
+     * @return task date description
+     */
+    public String getTask() {
+        return description + "(" +word+ ": " +date+ " " +time +")";
+    }
+
```

Comment:
You could try implementing a normal overriden toString method in your task class then override it for each subclass